templates/survey-wizard/index.qmd

```diff
@@ -45,7 +46,10 @@ In this example, we use the `shiny_validate` package to validate the form inputs
 **Packages:**
 
 * `pandas`
```

Comment:
For consistency, can we also link to pandas's website? `https://pandas.pydata.org/` or `https://pandas.pydata.org/docs/`

Reply:
I'm going to defer in favor of doing a whole pass through templates (#240)